js/piwik.js

```diff
@@ -1403,7 +1403,17 @@ if (typeof Piwik !== 'object') {
             },
             hasNodeCssClass: function (node, klassName)
             {
-                if (node && klassName && node.className) {
+                if(!node || !klassName) {
+                    return false;
+                }
+
+                // For modern browsers cf http://caniuse.com/#search=classList
+                if(node.classList) {
+                    return node.classList.contains(klassName);
```

Comment:
We're using saucelabs partially for manual testing but we'd need to run them automated on each commit via travis. This would be doable but it'd need to be implemented / configured to run them automatically via saucelabs (or browserstack). 

Also a problem is, I think, that some of the tests currently do not work eg on IE6. Not because `piwik.js` is not compatible with IE6 but I think some tests aren't.

So those are the two problems so far